pages/platform/kubernetes-k8s/upgrading-kubernetes-version/guide.en-asia.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Upgrading Kubernetes version on an OVH Managed Kubernetes cluster'
+slug: upgrading-kubernetes-version
+excerpt: ''
+section: 'User guides'
+---
+
+<style>
+ pre {
+     font-size: 14px;
+ }
+ pre.console {
+   background-color: #300A24; 
+   color: #ccc;
+   font-family: monospace;
+   padding: 5px;
+   margin-bottom: 5px;
+ }
+ pre.console code {
+   border: solid 0px transparent;
+   font-family: monospace !important;
+   font-size: 0.75em;
+   color: #ccc;
+ }
+ .small {
+     font-size: 0.75em;
+ }
+</style>
+
+**Last updated September 5th, 2019.**
+
+## Objective
+
+OVH Managed Kubernetes service provides you Kubernetes clusters without the hassle of installing or operating them. In this guide we cover how you can easily upgrade your cluster to the next minor version of Kubernetes.
+
+On OVH Managed Kubernetes we have deployed a *Upgrade to the next minor version of Kubernetes* feature to upgrade your cluster from one version to the next (*e.g.* 1.13 to 1.14).
+
+## Requirements 
+
+
+- An OVH Managed Kubernetes cluster
+
+## Important information on the upgrading process
+
+Before you launching the upgrade, please back up your cluster’s configuration, and check that your worker tools are compatible with this new version.
+
+The upgrading process will try to do a [rolling upgrade](https://en.wikipedia.org/wiki/Rolling_release). Please ensure that your cluster includes enough worker nodes to allow for a rolling upgrade. 
+
+To minimise downtime for your pods and your overall service, we will drain each of your nodes, only moving on to the next node when the current one is functional (i.e. when the status changes to ‘Ready’). Your admin components and server API will also be updated.
+
+ This operation usually takes around 10 minutes, depending on your cluster type and size.
```

Comment:
Maybe we could precise that it takes around 5mn per node ?

Reply:
Yeah, good idea...